plugins/inputs/ravendb/ravendb.go

```diff
@@ -378,52 +375,59 @@ func gatherServer(r *RavenDB, acc telegraf.Accumulator) {
+		"cpu_process_usage":                                             serverResponse.Cpu.ProcessUsage,
+		"cpu_machine_usage":                                             serverResponse.Cpu.MachineUsage,
+		"cpu_processor_count":                                           serverResponse.Cpu.ProcessorCount,
+		"cpu_assigned_processor_count":                                  serverResponse.Cpu.AssignedProcessorCount,
+		"cpu_thread_pool_available_worker_threads":                      serverResponse.Cpu.ThreadPoolAvailableWorkerThreads,
+		"cpu_thread_pool_available_completion_port_threads":             serverResponse.Cpu.ThreadPoolAvailableCompletionPortThreads,
+		"memory_allocated_in_mb":                                        serverResponse.Memory.AllocatedMemoryInMb,
+		"memory_installed_in_mb":                                        serverResponse.Memory.InstalledMemoryInMb,
+		"memory_physical_in_mb":                                         serverResponse.Memory.PhysicalMemoryInMb,
+		"memory_low_memory_severity":                                    mapMemorySeverity(serverResponse.Memory.LowMemorySeverity),
+		"memory_total_swap_size_in_mb":                                  serverResponse.Memory.TotalSwapSizeInMb,
+		"memory_total_swap_usage_in_mb":                                 serverResponse.Memory.TotalSwapUsageInMb,
+		"memory_working_set_swap_usage_in_mb":                           serverResponse.Memory.WorkingSetSwapUsageInMb,
+		"memory_total_dirty_in_mb":                                      serverResponse.Memory.TotalDirtyInMb,
+		"disk_system_store_used_data_file_size_in_mb":                   serverResponse.Disk.SystemStoreUsedDataFileSizeInMb,
+		"disk_system_store_total_data_file_size_in_mb":                  serverResponse.Disk.SystemStoreTotalDataFileSizeInMb,
+		"disk_total_free_space_in_mb":                                   serverResponse.Disk.TotalFreeSpaceInMb,
+		"disk_remaining_storage_space_percentage":                       serverResponse.Disk.RemainingStorageSpacePercentage,
+		"license_type":                                                  serverResponse.License.Type,
+		"license_utilized_cpu_cores":                                    serverResponse.License.UtilizedCpuCores,
+		"license_max_cores":                                             serverResponse.License.MaxCores,
+		"network_tcp_active_connections":                                serverResponse.Network.TcpActiveConnections,
+		"network_concurrent_requests_count":                             serverResponse.Network.ConcurrentRequestsCount,
+		"network_total_requests":                                        serverResponse.Network.TotalRequests,
+		"network_requests_per_sec":                                      serverResponse.Network.RequestsPerSec,
+		"cluster_node_state":                                            mapNodeState(serverResponse.Cluster.NodeState),
```

Comment:
Please keep the string here instead of mapping it to a cryptic number. A sting is much easier to discover during data-analysis especially by people that don't know this code.

Reply:
done

Reply:
Please mark as resolved.